go/vt/vtctl/vtctlclient/wrapper.go

```diff
@@ -27,10 +27,14 @@ import (
 	logutilpb "vitess.io/vitess/go/vt/proto/logutil"
 )
 
+var (
+	defaultTimeout = time.Hour
```

Comment:
Maybe make this a flag?

Reply:
My intent was to use any configured timeout on the context as the way a user would specify a timeout, however to limit the possibility of breakage I kept the one hour timeout that was hard coded before in the event someone passed in a Background or TODO context.

Reply:
I looked at all the callsites for RunCommandAndWait, and they're all sending in `context.TODO`. So, I think it's better to leave this as is for now.

Ideally, we should fix those call sites eventually.